### ISSUE TITLE
Date format is confusing for US users

### ISSUE THREAD

**Issue by arcalinea** [matrix-org/cerulean#45]
The date format is DD/MM/YYYY, which threw me off. If I submitted a PR changing it to "Jan 25, 2020" format, would that be welcome?

**Comment by scottwallacesh** [comment]
`YYYY-MM-DD` would be a more ISO compliant format.

**Comment by roke-julian-lockhart** [comment]
Indeed, @scottwallacesh, ISO 8601 should be the fallback format when localization is unavailable, but this should be localized.
